test/Microsoft.AspNet.Mvc.TagHelpers.Test/TextAreaTagHelperTest.cs

```diff
@@ -46,32 +47,32 @@ public class TextAreaTagHelperTest
                 {
                     { null, typeof(Model), null,
                         new NameAndId("Text", "Text"),
-                        Environment.NewLine + "HtmlEncode[[]]" },
+                        Environment.NewLine },
```

Comment:
Should have absolutely no encoding differences from previous tests. This indicates an unexpected behaviour change.